tools/booster/Stats.hs

```diff
@@ -43,13 +43,13 @@ instance (Floating a, PrintfArg a, Ord a) => Pretty (RequestStats a) where
                 <+> withUnit stats.average
                 <+> parens ("+-" <+> withUnit stats.stddev)
                 <> ", range"
-                    <+> brackets (withUnit stats.minVal <> ", " <> withUnit stats.maxVal)
+                <+> brackets (withUnit stats.minVal <> ", " <> withUnit stats.maxVal)
             , "Total time in kore-rpc code:"
                 <+> withUnit stats.koreTotal
             , "Average time per request in kore-rpc code:"
                 <+> withUnit stats.koreAverage
                 <> ", max"
-                    <+> withUnit stats.koreMax
+                <+> withUnit stats.koreMax
```

Comment:
This was added by fourmolu-0.14.0.0, which I am supposed to use since the upgrade.

Reply:
... and now CI's formatting run removed it :exploding_head: :man_shrugging: